var/spack/repos/builtin/packages/cudnn/package.py

```diff
@@ -12,6 +12,22 @@ class Cudnn(Package):
 
     homepage = "https://developer.nvidia.com/cudnn"
 
+    version('7.5.1-10.1-x64', '24017f4a56159d48fd5a31c8a930167b',
+            url='https://developer.download.nvidia.com/compute/redist/cudnn/v7.5.1/cudnn-10.1-linux-x64-v7.5.1.10.tgz')
+    version('7.5.1-10.1-ppc64le', '',
+            url='https://developer.download.nvidia.com/compute/redist/cudnn/v7.5.1/cudnn-10.1-linux-ppc64le-v7.5.1.10.tgz')
+    version('7.5.1-10.0-x64', '703ed4be4d242ff4bc0ca48aaf2029bc',
+            url='https://developer.download.nvidia.com/compute/redist/cudnn/v7.5.1/cudnn-10.0-linux-x64-v7.5.1.10.tgz')
+    version('7.5.1-10.0-ppc64le', '8348cbab01970b75836d945d8a75915e',
+            url='https://developer.download.nvidia.com/compute/redist/cudnn/v7.5.1/cudnn-10.0-linux-ppc64le-v7.5.1.10.tgz')
+    version('7.5-10.1-x64', '4edc7fcfeada9f2363e97f9875154e8f',
```

Comment:
These should be 7.5.0. I know the version scheme here is awful, but we can make it slightly more consistently awful.